quinn-proto/src/connection/mtud.rs

```diff
@@ -143,18 +152,18 @@ impl MtuDiscovery {
     ///
     /// Calling this function will close the previous loss burst. If a black hole is detected, the
     /// current MTU will be reset to `BASE_UDP_PAYLOAD_SIZE`.
-    pub(crate) fn detect_black_hole(&mut self, now: Instant) -> bool {
-        if self.black_hole_detector.detect_black_hole() {
-            self.current_mtu = BASE_UDP_PAYLOAD_SIZE;
+    pub(crate) fn black_hole_detected(&mut self, now: Instant) -> bool {
+        if !self.black_hole_detector.black_hole_detected() {
+            return false;
+        }
```

Comment:
Looks like this refactoring (incl. rename) landed in the wrong commit.

Reply:
I will move it to the original commit introducing robust black hole detection